fixtures/src/connect_to_db.py

```diff
@@ -0,0 +1,6 @@
+import psycopg2
+from src.config import dbname, user, password, host
```

Comment:
I'd call the file differently, like helpers.py or anything else, but whatever, let's leave it like this. We'll change it if we need to actually extend this script at any time in the future.

Reply:
I would even move the config files to separate directory because there are three config files and we keep the actual code on the same level which was pretty confusing to me.

Reply:
The config approach is wrong, we should keep these values as environment variables and have one file that would just use these variables instead of having multiple files for multiple environments, I'll create an issue for fixing that.